src/Hashing/ArgonHashing.php

```diff
@@ -57,22 +57,22 @@ class ArgonHashing extends AbstractHashing
     /**
      * __construct.
      *
-     * @param (array) $options
+     * @param array $options
      *
      * @since 3.0.0
      */
-    public function __construct(array $options)
+    public function __construct(array $options = [])
```

Comment:
The `$options` should be the empty array by default, not null because the type hint is array.

Reply:
yes you're right